### DISCUSSION TITLE
购买专业版后仍然显示免费版

### DISCUSSION THREAD

**Discussion by machaogoodluck** [getlantern/lantern-forum-cn#181] [Q&A]
尊敬的管理员您好，我已经按照您的要求下载最新版本，而且我的账户在购买专业版后还是显示免费版，目前已经不能正常上网，我的购买日期为2023年11月17日，服务时长为两年，希望尽快得到您的答复。账户名称machaogoodluck@163.com
![Screenshot_20240212_141431](https://github.com/getlantern/lantern-forum-cn/assets/161196556/8ad82016-ecfe-4fb5-ac06-f650eea9df68)

**Comment by MaxMKMax** [comment]
@machaogoodluck 谢谢您的反馈. 好的明白了! 我帮您和团队说明这个问题。我帮你紧急解决问题。让我通知技术团队!

**Reply by machaogoodluck** [reply]
目前还是免费版，希望尽快解决，如果不能解决的话，希望剩余时长能够退款

**Reply by MaxMKMax** [reply]
@machaogoodluck 您好. 我们会给您延长专业版时间. 您放心! 谢谢您的支持.

**Reply by MaxMKMax** [reply]
@machaogoodluck 您好。已经更新了您的专业计划并为您重新开始您的开始日期。这是代码：DR9HD-HBF9J-K3JFX-QH2VD-BKDGF

您也可以通过此链接找到完成操作的说明和方法: https://lantern.io/zh-Hans/faq

**Reply by machaogoodluck** [reply]
管理员您好，蓝灯每一次启动后还是显示免费版，需要手动输入以上激活码后才会变成专业版，但是变为专业版后还是不能正常上网，不能连接，访问网站总是显示没有连接互联网，拜托尽快解决以上问题，目前使用其他软件才能正常上网

**Reply by MaxMKMax** [reply]
@machaogoodluck 好的。已收到您的反馈和建议。我会让用户体验团队了解这个问题，我们将对其进行优化。

**Reply by machaogoodluck** [reply]
管理员您好，变为专业版后显示连接上，还是不能正常上网，访问网站根本打不开，目前使用其他付费软件才能正常上网，拜托尽快解决以上问题，如果真的不能解决，希望退款，不要再消耗一个蓝灯老用户的耐心

**Reply by machaogoodluck** [reply]
专业版连接不上，按照你们的要求，安装了最新版本，启动后又变回了免费版，而且不能打开任何网站

**Reply by MaxMKMax** [reply]
@machaogoodluck 您好. 我帮您重新申请一个激活码了. 等待支付团队回复我. 请您等等啊! 
请留意邮件或者GitHub, 我们会发给您.
谢谢您的耐心等待, 也谢谢您支持蓝灯!
